track_and_trace/README.md

```diff
@@ -0,0 +1,34 @@
+# Track and Trace
+
+The Grid Track and Trace transaction family allows users to track goods as they
+move through a supply chain. Records for goods include a history of ownership
+and custodianship, as well as histories for a variety of properties such as
+temperature and location. These properties are managed through a 
+user-specifiable system of record types. This README outlines how to start an
+instance of Track and Trace locally.
+
+### Prerequisites
+
+Running Track and Trace requires [docker](https://www.docker.com/get-started) to
+be installed locally on your machine.
+
+### Getting Started
+
+1) Clone the [grid](https://github.com/hyperledger/grid) and
+[grid-contrib](https://github.com/hyperledger/grid-contrib) repositories from github.
+    ```
+    $ git clone https://github.com/hyperledger/grid.git
+    $ git clone https://github.com/hyperledger/grid-contrib.git
+    ```
+2) Navigate to the Track and Trace directory in the grid-contrib repository and run docker-compose.
```

Comment:
This line is no longer accurate with the change to move to the grid repo instead of grid-contrib